packages/quantic/force-app/main/default/lwc/quanticQueryError/quanticQueryError.html

```diff
@@ -0,0 +1,36 @@
+<template>
+    <template if:true={hasError}>
+        <div class="slds-grid slds-wrap">
+            <div class="slds-col slds-size_12-of-12">
+                <h3 class="slds-text-heading_medium slds-text-color_error slds-text-align_center slds-text-title_bold slds-var-m-around_small">{errorTitle}</h3>
+                <p class="slds-text-align_center">{description}</p>
+            </div>
+            <template if:true={link}>
+                <div class="slds-col slds-size_12-of-12">
+                  <a class="slds-m-vertical_x-small slds-align_absolute-center" href={link}>Coveo Online Help</a>  
+                </div>
+            </template>
+            <template if:false={link}>
+                <template if:false={showMoreInfo}>
+                    <div class="slds-m-vertical_x-large slds-align_absolute-center">
+                        <lightning-button variant="brand" label="More Information"  class="slds-m-around_x-small" onclick={onClick}></lightning-button>
```

Comment:
I'd suggest to name the event handler `onShowMoreInfoClick` or `handleShowMoreInfoClick`. It'll be easier to know what the handler does in the component code.